backend/db/seeders/20240522125843-seed-spots.js

```diff
@@ -14,11 +19,9 @@ module.exports = {
       lng: -122.4194,
       name: 'Wonderful Spot',
       description: 'A beautiful spot in the hear of the city.',
-      price: 100.00
-      previewImage: ,
-      avgRating: 5.5,
-      createdAt: new Date(),
-      updatedAt: new Date()
+      price: 100.00,
+      previewImage: images/image1.jpg,
```

Comment:
Reference what this type of data is supposed to be in your db diagram

Reply:
When I view my DB Diagram . its showing that this type of data should be datetime

Reply:
Line 23? The "previewImage"?

Reply:
Looking at my db digram . previewimage is not under spots

Reply:
Ahh so you do not want to have `previewImage` in the spots seeder, migration, or the models. right?

Reply:
No I do want it, because based of the API docs it appears that its needed

Reply:
Nevermind! I think its best i remove it